pos_restrict_price_discount_buttons/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "POS Restrict Price Discount Buttons",
+    "summary": "POS Restrict Price Discount Buttons",
```

Comment:
Restrict access to 'Price' and 'Discount' buttons in POS

Reply:
Resolved